### ISSUE TITLE
Accessibility: Contrast of red-on-red warning (e.g. must allow requests for access)

### ISSUE THREAD

**Issue by adam3smith** [QualitativeDataRepository/dataverse#114]
Tested with WAVE & manually

On a project with restricted files and without terms of access ([like this one](https://data.stage.qdr.org/dataset.xhtml?persistentId=doi:10.33564/FK2RIYZ9U)), try to disable "Request Acess" and Save. The red warning doesn't have sufficient contrast

Rust tweaking the two reds a bit would suffice, eg. Make
Foreground: #AA4241
Background: #F7EDED
gets us >5
(I don't think we're using these elsewhere, so don't need to blend with other colors
